File: modules/azule_codesign.py
#!/usr/bin/env python

import plistlib
import sys

file = open(sys.argv[1], 'rb')
data = plistlib.load(file)

for x, y in dict(data).items():
	if type(y) is dict:
		for key, val in dict(y).items():
			if 'PlugIns' in key or 'WatchApp.app' in key:
				del data[x][key]
	else:
		for val in list(y):
			if 'PlugIns' in val or 'WatchApp.app' in val:
				data[x].remove(val)

file = open(sys.argv[1], "wb")
plistlib.dump(data, file)

File: modules/azule_codesign.js
#!/usr/bin/env node

const fs = require('fs');
const plist = require('plist');
const args = process.argv.slice(2);

try {
  const data = fs.readFileSync(args[0], 'utf8');
  var json = plist.parse(data);
  for (let x in json) {
    if (Array.isArray(json[x])) {
      for (let [i, y] of json[x].entries()) {
        if (y.includes('PlugIns') || y.includes('WatchApp.app')){
          delete json[x][i];
        }
      }
    }
    else {
      for (let y in json[x]) {
        if (y.includes('PlugIns') || y.includes('WatchApp.app')){
          delete json[x][y];
        }
      }
    }
  }
  fs.writeFileSync(args[0], plist.build(json));
}
catch(err) {
  console.log(err)
}
